Makefile

```diff
@@ -43,6 +44,7 @@ NODE_EXE = node$(EXEEXT)
 NODE ?= ./$(NODE_EXE)
 NODE_G_EXE = node_g$(EXEEXT)
 NPM ?= ./deps/npm/bin/npm-cli.js
+TEST_CI = ./tools/test-ci.sh
```

Comment:
If you made this something like:

```bash
TEST_CI = tools/test.py --mode=release -J
```

then wouldn't you get the same deduplication without the indirection of a separate file?